test/remote/gateways/remote_sum_up_test.rb

```diff
@@ -123,6 +123,26 @@ def test_failed_void_invalid_checkout_id
     assert_equal 'Resource not found', response.message
   end
 
+  def test_successful_refund
+    gateway = SumUpGateway.new(fixtures(:sum_up_testing))
```

Comment:
Maybe we can find a better name and include it in the fixtures.yml file, to keep in mind that we need to use this credentials as well to run remote test.